main/Resource.rdef

```diff
@@ -0,0 +1,129 @@
+
+resource mini_icon array {
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFADA"
+	$"FFFFFFFFFFFFFF0000000000FFFAF9FF"
+	$"FFFFFFFFFFFFFF0000190000FAF9FFFF"
+	$"FF20FFFFFFFFFF003F193FFAF9000000"
+	$"202020FFFFFF0B000019FAF9FF001500"
+	$"20202020F9F900003FFAF900FF001500"
+	$"FF2020F9F9F9F900FAF900000B001500"
+	$"FFFF20F9F9F9F9FAF93F3F3F00001500"
+	$"FFFFFF0BF9F940103F403F3F3F000000"
+	$"FFFFFF0000003F40403F3F403F3F3F00"
+	$"FFFFFF003F00003F403F3F403F3F000B"
+	$"FFFFFFFF003F00003F403F3F00000BFF"
+	$"FFFFFFFFFF00003F003F4000400000FF"
+	$"FFFFFFFFFFFF000000000000000000FF"
+	$"FFFFFFFFFFFFFFFFFF0B0BFFFFFFFFFF"
+};
+
+resource(1, "BEOS:M:application/x-be-resource") #'MICN' array {
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+	$"FFFFFFFFFFFFFF0000000000FFFFFFFF"
+	$"FFFFFFFFFFFFFF0000190000FFFFFFFF"
+	$"FF20FFFFFFFFFF003F193F00FF000000"
+	$"202020FFFFFF0B0000190000FF001400"
+	$"20202020F9F900003F193F00FF001400"
+	$"FF2020F9F9F9F900000000000B001400"
+	$"FFFF20F9F9F9F9403F3F3F3F00000000"
+	$"FFFFFF0BF9F9403F3F403F3F3F00000B"
+	$"FFFFFF0000003F40403F3F403F3F3F00"
+	$"FFFFFF003F00003F403F3F403F3F000B"
+	$"FFFFFFFF003F00003F403F3F00000BFF"
+	$"FFFFFFFFFF00003F003F4000400000FF"
+	$"FFFFFFFFFFFF000000000000000000FF"
+	$"FFFFFFFFFFFFFFFFFF0B0BFFFFFFFFFF"
+};
+
+resource file_types message {
```

Comment:
Please move lines 1-39 to just under line 60.  Also you'll need to update main/makefile to point to the rdef instead of the rsrc file.